pkg/imagestore/imagestore_test.go

```diff
@@ -195,3 +195,63 @@ var _ = Describe("HaveVersion", func() {
 		Expect(store.HaveVersion("4.8", "aarch64")).To(BeFalse())
 	})
 })
+
+var _ = Describe("NewImageStore", func() {
+
+	It("should error when RHCOS_IMAGES are not set i.e. versions is an empty slice", func() {
```

Comment:
I think you should add one happy path scenario to help us prevent regressions on the NewImageStore specifically.

Reply:
Done :heavy_check_mark: